Frontend/src/data/rental/onSave.js

```diff
@@ -81,7 +81,14 @@ export async function onReturnAndSave(context, employee) {
 }
 
 export default async function onSave(context) {
-  const { doc, closePopup, createNew, contextVars } = context;
+  const { doc, closePopup, createNew, contextVars, form } = context;
+
+  if (!form.wasChecked && !form.checkValidity()) {
+    form.wasChecked = true;
+    notifier.warning('Einige benötigte Felder sind nicht (korrekt) ausgefüllt. Trotzdem speichern?');
```

Comment:
will this also highlight the fields that are still missing?

Reply:
All fields that are in whatever way invalid (empty but required, invalid pattern, ...) are highlighted in light red.